sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAIClientBuilderTest.java

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.*;
```

Comment:
I think you will need to expand the star imports, as the style check script will probably not like it